### PR TITLE
Added Animations

### PR DESCRIPTION
**Pull request by mitul-bhatia**
Thank you for making a pull request to ZTM's Animation Nation project. We will review your pull request as soon as we can, but keep in mind we can get very busy during Hacktoberfest.

While you're waiting, please look over this checklist. Feel free to write a description of what you did as well!

## What I did

i added my animation to it , it looks ashetic and is good to look

## Pull request checklist

- [ ] This pull request uses **HTML and CSS** only and provides a `meta.json` file.
- [ ] All files in this pull request are in a directory named: `<github_username>-<art_name>` without the <>.
- [ ] Your pull request contains `index.html` and `styles.css`.
	- [ ] Your html file is named exactly: `index.html`
	- [ ] Your css file is named exactly: `styles.css`
- [ ] Your pull request your `meta.json` with:
	- [ ] `githubHandle`: being your unique github user name,
	- [ ] `artName` : name of your animation
- [ ] Your submission includes **at least one animation**.
- [ ] Your pull request **does not modify any other files** in the repository.

#### Not normally needed
- [ ] Check this if you would like to discuss an improvement besides adding animations

**Note: if your pull request makes an improvement besides adding a CSS animation you will need to discuss this with the maintainers. You can help kickstart this process by explaining what your change does and why you are making it in the "What I did" section**

## What to expect

- If your pull request meets the checklist requirements it will be approved and merged by a maintainer
- If your pull request needs some changes. A maintainer will leave a comment requesting changes
- If your pull request significantly violates the criteria in the checklist it will be closed with feedback.

### CODE REVIEW

**Comment by mitul-bhatia** [comment]
please accept it

File: Art/mitulBhatia-abs/icon.png
Comment (LaurelineP) [inline Art/mitulBhatia-abs/icon.png:1]: # Unnecessary picture
- [ ] Please remove the icon as this is generate when your PR is accepted automatically

File: Art/mitulBhatia-abs/index.html
Comment (LaurelineP) [inline Art/mitulBhatia-abs/index.html:1]: # Unnecessary comments
- [ ] Please remove the comments

**Review by LaurelineP — CHANGES_REQUESTED** [review]
Aloha @mitul-bhatia 
Thanks for your contribution.
Please address the given feedback and the following
- [ ] the start of your foldername  should be the github handle ( you've added camelcase which does not match your github handle ) 
Please adjust

**Comment by mitul-bhatia** [comment]
@LaurelineP i made all changes that were required now please accept it

**Comment by LaurelineP** [comment]
> @LaurelineP i made all changes that were required now please accept it

There is still the file icon to be removed, please address this

**Comment by LaurelineP** [comment]
For maintaining reasons, your PR is set to stale.
In the next 2 following days this will be closed if the given feedbacks and/or your PR is not correct.
@mitul-bhatia Please address the given feedbacks for your PR to be merged

**Pull request merged**